src/asvz_bot.py

```diff
@@ -408,46 +426,64 @@ def __get_enrollment_and_start_time(driver):
 
     @staticmethod
     def __get_enrollment_time(driver):
-        enrollment_interval_raw = driver.find_element(
-            By.XPATH, "//dl[contains(., 'Einschreibezeitraum')]/dd"
-        )
-        # enrollment_interval_raw is like 'So, 09.05.2021 06:35 - Mo, 10.05.2021 07:05'
-        enrollment_start_raw = (
-            enrollment_interval_raw.get_attribute("innerHTML")
-            .split("-")[0]
-            .split(",")[1]
-            .strip()
-        )
         try:
-            enrollment_start = datetime.strptime(enrollment_start_raw, "%d.%m.%Y %H:%M")
-        except ValueError as e:
-            logging.error(e)
-            raise AsvzBotException(
-                "Failed to parse enrollment start time: '{}'".format(
-                    enrollment_start_raw
-                )
+            enrollment_interval_raw = driver.find_element(
+                By.XPATH, "//dl[contains(., 'Einschreibezeitraum')]/dd"
+            )
+            # enrollment_interval_raw is like 'So, 09.05.2021 06:35 - Mo, 10.05.2021 07:05'
+            enrollment_start_raw = (
+                enrollment_interval_raw.get_attribute("innerHTML")
+                .split("-")[0]
+                .split(",")[1]
+                .strip()
             )
+            try:
+                enrollment_start = datetime.strptime(enrollment_start_raw, "%d.%m.%Y %H:%M")
+            except ValueError as e:
+                logging.error(e)
+                raise AsvzBotException(
+                    "Failed to parse enrollment start time: '{}'".format(
+                        enrollment_start_raw
+                    )
+                )
+        except NoSuchElementException as e:
+            logging.info("No enrollment time found. Assuming that enrollment is open.")
+            enrollment_start = datetime.today()
```

Comment:
I think this is a nice fallback option. However, I guess you encountered this case, if there was no `Einschreibezeitraum` for events? If that's the case, there would be a `Anmeldezeitraum` and we could check for this as well (like you did later on with `Datum/Zeit`/`Lektionen`)

(https://schalter.asvz.ch/tn/lessons/560239)